tests/ignite/contrib/metrics/test_mae.py

```diff
@@ -0,0 +1,25 @@
+from ignite.contrib.metrics import MaximumAbsoluteError
+import torch
+
+
+def test_maximum_absolute_error(tol=1e-6):
+    a = torch.tensor([0.0, 0.2, -0.8])
+    b = torch.tensor([1.0, -2.0, 3.0])
+    c = torch.tensor([0.0, 2.0, 0.0])
+    d = torch.tensor([0.0, 0.0, -5.0])
+    ground_truth = torch.tensor([0.0, 0.0, 0.0])
+
+    m = MaximumAbsoluteError()
+    m.reset()
+
+    m.update((a, ground_truth))
+    assert abs(m.compute() - 0.8) < tol
```

Comment:
Please use pytest.approx instead of manual compare as it is done [here](https://github.com/pytorch/ignite/blob/4aa10a0e5345043ce2e3cc623f72a3537cc982d8/tests/ignite/metrics/test_mean_pairwise_distance.py#L21)